src/lib.rs

```diff
@@ -68,7 +68,7 @@ pub struct CharLS {
     decoder: Option<CharlsDecoder>,
 }
 
-#[derive(Default)]
+#[derive(Default, Debug)]
```

Comment:
I think it's also safe to derive `Clone` and `PartialEq`.

```suggestion
#[derive(Debug, Default, Clone, PartialEq)]
```

Reply:
Done